ui/qml/pages/PairSelectDeviceType.qml

```diff
@@ -15,7 +15,7 @@ PageListPL {
                     var pairpage = pageStack.push(Qt.resolvedUrl("./PairPage.qml"));
                     pairpage.deviceType = deviceType;})
             } else {
-                var pairpage = pageStack.push(Qt.resolvedUrl("./PairPage.qml"));
+                var pairpage = pageStack.push(Qt.resolvedUrl("/home/jf/git/harbour-amazfish/ui/qml/pages/PairPage.qml"));
```

Comment:
I guess this shoudnt be here :)

Reply:
Yes, sorry about that... I changed the path to debug an issue with the pairing page, and I shouldn't have committed that! I reverted that change.